src/coap_debug.c

```diff
@@ -789,7 +788,8 @@ coap_log_impl(coap_log_t level, const char *format, ...) {
 
     coap_ticks(&now);
     if (print_timestamp(timebuf,sizeof(timebuf), now))
-      fprintf(log_fd, "%s ", timebuf);
+      fprintf(log_fd, "%s.%03u ", timebuf,
```

Comment:
This gives us milliseconds. Maybe use `coap_ticks_to_rt_us()` here as well?

Reply:
Oops - I did mean millisecs in the title - but can make it microsecs.

Would you prefer that print_timestamp() does the microsecs appendage?